src/test/java/net/helpscout/api/ApiClientTest.java

```diff
@@ -64,6 +77,33 @@ public void shouldReturnDetailedErrorMessageForBadRequest() {
         }
     }
     
+	@Test
+	public void shouldRetrieveSpecificCustomer() throws ApiException {
+		givenThat(get(urlEqualTo("/v1/customers/60984612.json"))
+				.willReturn(aResponse().withStatus(HTTP_OK)
+				.withBody(getResponse("customer"))));
```

Comment:
I think we should be able to just use `withBodyFile` and eliminate the `getResponse` method.